### PR TITLE
tailscale: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/tailscale.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# tailscale
+
+> The easiest and most secure way to use WireGuard and 2FA.
```
Comment (marchersimon) [inline pages/common/tailscale.md:3]: Shouldn't we keep the descriptions as objective as possible?

File: pages/common/tailscale.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Connect to Tailscale:
+
+`sudo tailscale up`
+
+- Disconnect from Tailscale:
+
+`sudo tailscale down`
+
+- Display current Tailscale IP addresses:
```
Comment (marchersimon) [inline pages/common/tailscale.md:14]:
```suggestion
- Display the current Tailscale IP addresses:
```

File: pages/common/tailscale.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ping a peer node at the Tailscale layer and display which route it took for each response:
+
+`tailscale ping {{ip|hostname}}`
+
+- Analyze the local network conditions and display the result:
+
+`tailscale netcheck`
+
+- Run a web server for controlling Tailscale:
```
Comment (marchersimon) [inline pages/common/tailscale.md:26]: Does this start a local server and opens it in the web browser?

File: pages/common/tailscale.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/tailscale.md:3]: The service describes itself with a lot of cool adjectives lol
```suggestion
> A private WireGuard network service.
```

File: pages/common/tailscale.md (hunk 3 above)
Comment (navarroaxel) [reply pages/common/tailscale.md:26]: Just start the server, and displays the URL http://localhost:8088 to enter to it. And the user should use `Ctrl + C` to stop it.

Comment (marchersimon) [reply pages/common/tailscale.md:26]:
```suggestion
- Start a web server for controlling Tailscale:
```

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

File: pages/common/tailscale.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Analyze the local network conditions and display the result:
+
+`tailscale netcheck`
+
+- Start a web server for controlling Tailscale:
+
+`tailscale web`
+
+- Display a shareable identifier to help diagnose issues:
```
Comment (sbrl) [inline pages/common/tailscale.md:30]: Identifier? That sounds like an ID number. Does this upload a diagnostic report to a remote server or something and return the id?

Comment (navarroaxel) [reply pages/common/tailscale.md:30]: from <https://tailscale.com/kb/1080/cli/#bugreport>
> Bugreport makes it easier to report bugs to the Tailscale team by marking diagnostic logs with indicators to make triage easier.
> 
> If you encounter a connectivity issue, run tailscale bugreport on the device experiencing the issue at the time you encounter it. This command will print a random identifier into diagnostic logs, which you can share with our team.
> 
> Identifiers look like this:
> 
> $ tailscale bugreport
> BUG-1b7641a16971a9cd75822c0ed8043fee70ae88cf05c52981dc220eb96a5c49a8-20210427151443Z-fbcd4fd3a4b7ad94
> 
> This command shares no personally-identifiable information, and is unused unless you share the bug identifier with our team.

Comment (sbrl) [reply pages/common/tailscale.md:30]: Ah, I see! Thanks for explaining - that makes sense now :-)

**Review by sbrl — APPROVED** [review]

**Pull request merged**